src/main/java/edu/wisc/my/ltiproxy/dao/LTILaunchPropertyFileDao.java

```diff
@@ -5,12 +5,13 @@
 
 import com.fasterxml.jackson.core.JsonParseException;
 import com.fasterxml.jackson.databind.JsonMappingException;
+import com.google.common.collect.Multimap;
 
 public interface LTILaunchPropertyFileDao{
     
     public Map<String, String> getLaunchParameters(String key) throws JsonParseException, JsonMappingException, IOException;
     
-    public Map<String, String[]> getHeadersToReplace(String key) throws JsonParseException, JsonMappingException, IOException;
+    public Multimap<String, String> getHeadersToReplace(String key) throws JsonParseException, JsonMappingException, IOException;
```

Comment:
I could see that. I was thinking that since we distribute this as a war, the webservice api was the one to go off of for versioning. That being said, we could definitely go to 2, or I could easily revert the api change, since it's not essential to the bugfix